modules/expirychecker/src/Utilities.php

```diff
@@ -54,7 +12,7 @@ public static function haveSameDomain(
      **/
     public static function getUrlFromRelayState(string $relayState): string
     {
-        if (strpos($relayState, "http") === 0) {
+        if (str_starts_with($relayState, "http")) {
```

Comment:
Prettier!

Reply:
I appreciate IDE suggestions like this. Honestly don't know how people get by with simple editors.

Reply:
The difficulty is that is a PHP 8.x function. Some people haven't transitioned from 7.x yet.

Reply:
This isn't a library though, so it doesn't matter. Good to know, though, in case I encounter that in library code.